xcodeutil/testsummaries.go

```diff
@@ -6,9 +6,92 @@ import (
 	"strconv"
 	"time"
 
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-tools/go-xcode/plistutil"
 )
 
+// const ...
+const (
+	OldTestSummaries TestSummaryType = "OldTestSummaries"
+	NewTestSummaries TestSummaryType = "NewTestSummaries"
+)
+
+// TestSummaryType ...
+type TestSummaryType string
```

Comment:
the package name describes the context well, you can use simply: Type -> testsummary.Type looks good.